shotover-proxy/tests/kafka_int_tests/mod.rs

```diff
@@ -427,33 +521,65 @@ async fn cluster_2_racks_multi_shotover_with_one_shotover_down(#[case] _driver:
     // Wait for check_shotover_peers to start
     tokio::time::sleep(Duration::from_secs(15)).await;
 
-    // Kill one shotover node
-    tokio::time::timeout(
-        Duration::from_secs(10),
-        shotovers.remove(0).shutdown_and_then_consume_events(&[]),
-    )
-    .await
-    .expect("Shotover did not shutdown within 10s");
+    // Send some produce and consume requests
+    let connection_builder = KafkaConnectionBuilder::new(driver, "localhost:9192");
+    produce_consume_partitions3(&connection_builder, "partitions3_rf3", 1, 500).await;
```

Comment:
lets do the full cluster_test_suite here.

Reply:
updated